subscription-service/src/test/kotlin/com/egm/stellio/subscription/service/SubscriptionServiceTests.kt

```diff
@@ -87,7 +88,8 @@ class SubscriptionServiceTests : WithTimescaleContainer {
             entities = setOf(
                 EntityInfo(id = null, idPattern = null, type = "Beekeeper"),
                 EntityInfo(id = "urn:ngsi-ld:Beehive:1234567890".toUri(), idPattern = null, type = "Beehive")
-            )
+            ),
+            expiresAt = ZonedDateTime.parse("2050-12-12T08:33:38Z")
```

Comment:
I know that 2050 is far away in the future but you can do something more robust with, for instance: `Instant.now().atZone(ZoneOffset.UTC).plusDays(1)`